core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +267,37 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
I would agree with you if this thing were not under a `Player` interface, in which case maybe the client doesn't even have control... 

I am still hesitant to be honest, in one sense I agree with you.